### PR TITLE
tools/dm: update manage-task.md and add precheck.md

### PR DESCRIPTION
**Pull request by yikeke**
Via: https://github.com/pingcap/docs-cn/pull/1139 PTAL @lilin90 @GregoryIan @csuzhangxc

### CODE REVIEW

**Comment by yikeke** [comment]
> Please also help update the following three files that have links pointing to the original precheck section (/tools/dm/manage-task.md#precheck-the-upstream-mysql-instance-configuration):
> 
> * overview.md
> * practice.md
> * troubleshooting.md

Sure :)

File: tools/dm/practice.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ To detect possible errors of data synchronization configuration in advance, DM p
 - DM automatically checks the corresponding privileges and configuration while starting the data synchronization task.
 - You can also use the `check-task` command to manually precheck whether the upstream MySQL instance configuration satisfies the DM requirements.
 
-For details about the precheck feature, see [Precheck the upstream MySQL instance configuration](/tools/dm/manage-task.md#precheck-the-upstream-mysql-instance-configuration).
+For details about the precheck feature, see [Precheck the upstream MySQL instance configuration](/tools/dm/precheck.md#precheck-the-upstream-mysql-instance-configuration).
```
Comment (lilin90) [inline tools/dm/practice.md]:
```suggestion
For details about the precheck feature, see [Precheck the upstream MySQL instance configuration](/tools/dm/precheck.md).
```

File: tools/dm/troubleshooting.md [hunk 2]
```diff
@@ -28,7 +28,7 @@ However, you need to reset the data synchronization task in some cases. For deta
 
 For database related passwords in all the DM configuration files, use the passwords encrypted by `dmctl`. If a database password is empty, it is unnecessary to encrypt it. For how to encrypt the plaintext password, see [Encrypt the upstream MySQL user password using dmctl](/tools/dm/deployment.md#encrypt-the-upstream-mysql-user-password-using-dmctl).
 
-In addition, the user of the upstream and downstream databases must have the corresponding read and write privileges. Data Migration also [prechecks the corresponding privileges automatically](/tools/dm/manage-task.md#precheck-the-upstream-mysql-instance-configuration) while starting the data synchronization task.
+In addition, the user of the upstream and downstream databases must have the corresponding read and write privileges. Data Migration also [prechecks the corresponding privileges automatically](/tools/dm/precheck.md#precheck-the-upstream-mysql-instance-configuration) while starting the data synchronization task.
```
Comment (lilin90) [inline tools/dm/troubleshooting.md]:
```suggestion
In addition, the user of the upstream and downstream databases must have the corresponding read and write privileges. Data Migration also [prechecks the corresponding privileges automatically](/tools/dm/precheck.md) while starting the data synchronization task.
```

File: tools/dm/overview.md [hunk 3]
```diff
@@ -77,7 +77,7 @@ Before using the DM tool, note the following restrictions:
     - 5.5 < MySQL version < 5.8
     - MariaDB version >= 10.1.2
 
-    Data Migration [prechecks the corresponding privileges and configuration automatically](/tools/dm/manage-task.md#precheck-the-upstream-mysql-instance-configuration) while starting the data synchronization task using dmctl.
+    Data Migration [prechecks the corresponding privileges and configuration automatically](/tools/dm/precheck.md#precheck-the-upstream-mysql-instance-configuration) while starting the data synchronization task using dmctl.
```
Comment (lilin90) [inline tools/dm/overview.md]:
```suggestion
    Data Migration [prechecks the corresponding privileges and configuration automatically](/tools/dm/precheck.md) while starting the data synchronization task using dmctl.
```

File: tools/dm/practice.md (hunk 1 above)
Comment (lilin90) [reply tools/dm/practice.md]: I think linking to the precheck.md file is clear enough :)

File: tools/dm/overview.md (hunk 3 above)
Comment (yikeke) [reply tools/dm/overview.md]: Updated. PTAL.

**Pull request merged**